packages/proxy/src/proxy.ts

```diff
@@ -83,6 +83,8 @@ export const FORMAT_HEADER = "x-bt-stream-fmt";
 export const LEGACY_CACHED_HEADER = "x-cached";
 export const CACHED_HEADER = "x-bt-cached";
 
+export const PROVIDER_HEADER = "x-bt-provider";
```

Comment:
hmm actually now that i think about it, since `x-bt-endpoint-name` lets you filter to select this, maybe we should name it `x-bt-used-provider`?